app/models/concerns/cas_client_data.rb

```diff
@@ -0,0 +1,265 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CasClientData
```

Comment:
A possible future refactoring of this would be to move the site-specific stuff into drivers?

Reply:
Yes, though, realistically, I doubt we'll ever enable drivers based on installation.